lang/test/org/partiql/lang/syntax/SqlParserTest.kt

```diff
@@ -3092,4 +3096,96 @@ class SqlParserTest : SqlParserTestBase() {
 
         assertEquals(withoutSemicolon, withSemicolon)
     }
+
+    //****************************************
+    // LET clause parsing
+    //****************************************
+
+    private val projectX = PartiqlAst.build { projectList(projectExpr(id("x"))) }
+
+    @Test
+    fun selectFromLetTest() = assertExpression("SELECT x FROM table1 LET 1 AS A") {
+        select(
+            project = projectX,
+            from = scan(id("table1")),
+            fromLet = let(
+                letBinding(
+                    lit(ionInt(1)),
+                    "A")
+            )
```

Comment:
Nit:  put all on one line.  same for others below.

Reply:
Revised formatting